xiFarm.sol

```diff
@@ -7,40 +7,37 @@ import "./IBEP20.sol";
 //REVEL IN IT'S BADNESS.
 
 contract XiFarm is Context, Ownable {
-    
-  using SafeMath for uint256;
-  
-  address poohAddy = 0x9673C2196fCAe71bE87864CDb04aDc4644559e89;//POOHTEST ADDY
-  address XiAddy = 0x098b246bf19ba9C5aD9C5dD994815e1db444eaA6;//XITEST ADDY
-  
-  IBEP20 POOH = IBEP20(poohAddy);
-  IBEP20 Xi = IBEP20(XiAddy);
-  
-  uint public xiStakeRewardCap = 20000000 * (10 ** 18); //20% of TOTAL Xi Supply
-  uint public xiStakeTime = 96000; //TIME IN BLOCKS
-  uint public xiTokensLeft; // Tracks how many REWARD tokens are left in contract
-  
-  uint public blockReward = (xiTokensLeft/xiStakeTime); // Keeps track of how many tokens are rewarded "per block" by overall duration.
-  uint totalStaked = 0;
-  
-  mapping (address => uint256) public stakedBalance;
+    using SafeMath for uint256;
 
+    address poohAddy = 0x9673C2196fCAe71bE87864CDb04aDc4644559e89; //POOHTEST ADDY
+    address XiAddy = 0x098b246bf19ba9C5aD9C5dD994815e1db444eaA6; //XITEST ADDY
+
+    IBEP20 POOH = IBEP20(poohAddy);
+    IBEP20 Xi = IBEP20(XiAddy);
+
+    uint256 public xiStakeRewardCap = 20000000 * (10**18); //20% of TOTAL Xi Supply
+    uint256 public xiStakeTime = 96000; //TIME IN BLOCKS
+    uint256 public xiTokensLeft; // Tracks how many REWARD tokens are left in contract
+
+    uint256 public blockReward = (xiTokensLeft / xiStakeTime); // Keeps track of how many tokens are rewarded "per block" by overall duration.
+    uint256 totalStaked = 0;
+
+    mapping(address => uint256) public stakedBalance;
+
+    constructor() public {
+        xiTokensLeft = 20000000 * (10**18); // Set this to the number of REWARD tokens to be sent to the contract by owners.
+    }
 
-  constructor() public {
-            xiTokensLeft = 20000000 * (10 ** 18); // Set this to the number of REWARD tokens to be sent to the contract by owners.
-  }
     //Struct that keeps information about each users stake, mapped to with Stakes
     struct stakeInfo {

```

Comment:
Can we maybe combine turn this into a struct with all these details (not in array form)

Then we can have `Stakes` be a map from `address` => `StakeInfo[]` which I think is cleaner.